configure

```diff
@@ -2297,6 +2297,8 @@ set_flag_if_supported "-Wglobal-constructors"
 # unknown -Wno-xxx flags.
 set_flag_if_supported -Wno-pragma-pack
 
+set_flag_if_supported -Wno-address-of-packed-member
+
```

Comment:
I think it's better to fix code, than silence compiler globally. we care about alignment. compiler warns for some reason. @dreammaster any idea about code?

Reply:
Removed it for now.